physijs/README.md

```diff
@@ -0,0 +1,18 @@
+# PhysiJS Changes
+
+The files here are 99.9% the same as the original [`physijs`](https://github.com/chandlerprall/Physijs) code.
+There are a few small modifications to make it compatible with bundlers and work out-of-the-box with zero config:
+
+- accepting the `PhysijsWorker` constructor function as an argument instead of just using `new Worker(...)`
+  - this allows for custom workers, such as those of [`worker-loader`](https://github.com/webpack-contrib/worker-loader) and [`webworkify`](https://github.com/browserify/webworkify)
+  - it also means one no longer has to set `Physijs.scripts.worker`
+- `require`ing `ammo.js` directly in the `physijs_worker.js` file instead of using [`importScripts`](https://developer.mozilla.org/en-US/docs/Web/API/Web_Workers_API/Using_web_workers#Importing_scripts_and_libraries)
+  - this can only be done because of the extra functionalities we get from a bundler and its custom worker
+  - this means one no longer has to set `Physijs.scripts.ammo`
+- slightly change the `Ammo` code so it doesn't try to `require` anything in its Emscripten header
```

Comment:
should this link to Ammo and Emscripten? I decided not to link when I wrote this as that could get readers into a giant rabbit hole of unneeded complexity very fast (not to mention it wouldn't match the old version that was used to make the file here). But at the same time, this is more _contributor_ documentation than it is _user_ documentation, so that context may make sense...

Or maybe the link to my commit message is enough...

Reply:
think I'm gonna leave it out, at least for now, and just go with my original reasoning for leaving it out. don't want to hold this back simply because of this consideration